web/src/styles/base/global.scss

```diff
@@ -129,6 +129,10 @@ ol {
   transform: translateY(0%);
 }
 
+.text-normal-override {
```

Comment:
[sand] Can we just call this `text-normal`?

Reply:
I left a comment on the component itself - I think we might not need/want this class at all

Reply:
done